io/post_gfs.F90

```diff
@@ -156,11 +156,13 @@ subroutine post_run_gfs(wrt_int_state,mypei,mpicomp,lead_write,      &
             if(mype==0) print *,'af read_xml at fh00,name=',trim(filenameflat)
           else if(ifhr > 0) then
             filenameflat = 'postxconfig-NT.txt'
-            if(size(paramset)>0) then
+            if(associated(paramset) .and. size(paramset)>0) then
```

Comment:
The Fortran compiler specifications do not state that the tests in an if clause will be executed ini the order written. You need to do the following to be fully compliant:
```
if (associated(paramset)) then
  if (size(paramset)>0) then
```

Reply:
Actually the size(paramset) is 1 but associated(paramset) is false.